tests/cases/conformance/classes/propertyMemberDeclarations/definePropertyESNext.ts

```diff
@@ -0,0 +1,20 @@
+// @target: esnext
```

Comment:
You can just merge this with `definePropertyES5.ts` (and rename it?) and use this:
```ts
// @target: es5, esnext
```